sdk/tables/data-tables/src/utils/connectionString.ts

```diff
@@ -20,6 +23,9 @@ export function getClientParamsFromConnectionString(
   connectionString: string,
   options?: TableServiceClientOptions
 ): ClientParamsFromConnectionString {
+  if (connectionString.toLocaleLowerCase().indexOf("usedevelopmentstorage=true") !== -1) {
```

Comment:
If we have to support lower case too, storage-blob and storage-queue packages would fail with the emulator.

Reply:
@joheredi Can log an issue for this?

Reply:
Yes! I filed 2 issues. This is not only for the Emulator but any connection string should have case-insensitive keys

Storage: https://github.com/Azure/azure-sdk-for-js/issues/13173
Tables: https://github.com/Azure/azure-sdk-for-js/issues/13172

Reply:
Yeah you are right!